homeassistant/components/zwave_js/cover.py

```diff
@@ -79,17 +79,19 @@ async def async_set_cover_position(self, **kwargs: Any) -> None:
 
     async def async_open_cover(self, **kwargs: Any) -> None:
         """Open the cover."""
-        target_value = self.get_zwave_value("Open")
-        await self.info.node.async_set_value(target_value, PRESS_BUTTON)
+        target_value = self.get_zwave_value("targetValue")
+        await self.info.node.async_set_value(target_value, 99)
 
     async def async_close_cover(self, **kwargs: Any) -> None:
         """Close cover."""
-        target_value = self.get_zwave_value("Close")
-        await self.info.node.async_set_value(target_value, PRESS_BUTTON)
+        target_value = self.get_zwave_value("targetValue")
+        await self.info.node.async_set_value(target_value, 0)
 
     async def async_stop_cover(self, **kwargs: Any) -> None:
         """Stop cover."""
-        target_value = self.get_zwave_value("Open")
-        await self.info.node.async_set_value(target_value, RELEASE_BUTTON)
-        target_value = self.get_zwave_value("Close")
-        await self.info.node.async_set_value(target_value, RELEASE_BUTTON)
+        target_value = self.get_zwave_value("Open") or self.get_zwave_value("Up")
+        if target_value:
```

Comment:
It would be best to already when setting up the entity decide if we support stop or not. Then we should modify what we return in the `supported_features` entity property.

Reply:
Reading the Z-Wave spec, all motor classes should support a form of "stop". But class A is done so by issuing a command that is opposite of the current movement.

Reply:
I only have a cursory knowledge of what's in this code base to know whether we can identify if it's currently moving and issue the opposite position command.

I was only aware of this particular error because it was happening on my own system and was able to dive in and fix it for my own window blinds, haha